src/app/static/src/app/components/projectListPage.vue

```diff
@@ -95,7 +99,7 @@
                     // so take this.newRegion as the only region
                     regionNames.push(this.newRegion)
                 }
-                const project = new Project(this.newProject, regionNames);
+                const project = new Project(this.newProject, regionNames, null, this.baselineOptions);
```

Comment:
can we keep the nullable arg at the end of the method, so it can just be omitted?

Reply:
Yeah, I left it where it was to keep the two regions params together, but you're right it is neater to leave it to the end..